string camera_name
string oid_name
---
bool success
string message